frontend/src/pages/ServiceDetails/ServiceDetailsPage.tsx

```diff
@@ -66,10 +66,10 @@ class ServiceDetails extends React.Component<ServiceDetailsProps, ServiceDetails
   constructor(props: ServiceDetailsProps) {
     super(props);
     const urlParams = new URLSearchParams(this.props.location.search);
-    const cluster = urlParams.get('cluster');
+    const cluster = urlParams.get('cluster') || HomeClusterName;
```

Comment:
`HomeClusterName` is not valid anymore. If the frontend does not provide cluster as a query param, then the backend can check for this and assume that you want the home cluster to keep backward compatability. Perhaps the backend should even throw an error if you're in "multicluster" mode and don't provide cluster as a query param but singlecluster should work the same in that you can use the same backend api calls without the cluster param.

Reply:
this is done as 'cluster' is a mandatory attribute, similar to other places we have, agree that this is not the best solution, and we need a separate PR for resolving them at all and 'HomeClusterName' is a keyword where to look at.
When the value is empty, the links generated does not have a 'cluster' param.